src/textual_summary/empty_group.jsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+export default function EmptyGroup(props) {
+  return (
+    <table className="table table-bordered table-striped table-summary-screen">
```

Comment:
How about using patternfly tables? https://rawgit.com/patternfly/patternfly-react/gh-pages/index.html?selectedKind=Table&selectedStory=PatternFly%20Table%20Styles&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

Reply:
I can do it, but not in the first implementation. My goal in this PR is to create React components from HAML files.